app/frontend/src/features/auth/useAuthStyles.ts

```diff
@@ -65,19 +80,59 @@ const useAuthStyles = makeStyles((theme) => ({
   },
   feedbackMessage: {
     textAlign: "center",
+    [theme.breakpoints.up("md")]: {
+      textAlign: "left",
+    },
   },
   form: {
     display: "flex",
     flexDirection: "column",
     marginBottom: theme.spacing(5),
     width: "100%",
+    [theme.breakpoints.up("md")]: {
+      marginBottom: 0,
+    },
   },
   formField: {
     marginBottom: theme.spacing(2),
+    [theme.breakpoints.up("md")]: {
+      marginBottom: theme.spacing(4),
+    },
   },
   formLabel: {
     color: theme.palette.text.primary,
     fontWeight: 700,
+    [theme.breakpoints.up("md")]: {
+      marginBottom: theme.spacing(2),
+    },
+  },
+  formWrapper: {
+    [theme.breakpoints.up("md")]: {
+      backgroundColor: "#f8f7f4",
```

Comment:
This isn't in the palette presumably. 🤔 Can you put /// TODO: Check this with the palette